plugins/template-validator.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const validator = require('screwdriver-template-validator');
+
+exports.register = (server, options, next) => {
+    server.route({
+        method: 'POST',
+        path: '/validator/template',
```

Comment:
Point of feedback: What does the endpoint for the template validator look like?

Alternative suggestions were:

1. Overload the `/validator` endpoint

To distinguish between validating a `sd-template.yaml` and a standard `screwdriver.yaml`, the caller could pass a parameter to distinguish a type. For instance, it could be `/validator?type=template` to communicate that the caller wants to validate a template.

This felt too ambiguous and overloading an endpoint for different configuration types could lead to confusion on the part of the user.

2. Use a `/validator/<type>` endpoint

This is the current implementation. If this is the preferred route, we would want to revisit the current `/validator` endpoint and refactor it. It could be refactored into `/validator/screwdriver`, `/validator/user`, or whatever type makes sense to describe it.